src/systempolicy/observability.go

```diff
@@ -103,7 +106,11 @@ func GetSystemObsData(wpfs types.WorkloadProcessFileSet) (opb.Response, error) {
 	return opbSysObsResponse, nil
 }
 
-func ClearSysDb(wpfs types.WorkloadProcessFileSet, duration int64) error {
-	err := libs.ClearWPFSDb(CfgDB, wpfs, duration)
+func ClearSysDb(wpfs types.WorkloadProcessFileSet, durationStr string) error {
+	if durationStr == "0" {
+		return errors.New("not a valid duration")
+	}
+	duration, _ := time.ParseDuration(durationStr)
```

Comment:
Please handle this error.
```suggestion
	duration, err := time.ParseDuration(durationStr)
	if err != nil {
	     return err
	}
```